webpack.config.js

```diff
@@ -211,6 +229,13 @@ if ( calypsoEnv === 'development' ) {
 	webpackConfig.plugins = webpackConfig.plugins.concat( [
 		new webpack.HotModuleReplacementPlugin(),
 		new webpack.LoaderOptionsPlugin( { debug: true } ),
+		new AutoDllPlugin( {
+			debug: true,
+			filename: '[name].js',
```

Comment:
we'll still want a hash in here

Reply:
its better to not use hashes during dev (speed/memory reasons).  then use hashes in production builds.

ddl plugin would just be for dev builds

Reply:
how much better? 

Ideally we wouldn't split dev and prod quite that much. Too easy to fall into a trap where dev works and prod bombs.